python/ray/dag/compiled_dag_node.py

```diff
@@ -1542,6 +1562,18 @@ def run(self):
         monitor.start()
         return monitor
 
+    def raise_if_too_many_inflight_requests(self):
+        num_in_flight_requests = self._execution_index - self._max_execution_index
```

Comment:
The name `_max_execution_index` seems a bit odd to me. Without reading the comments in the constructor, I would assume `_max_execution_index` should be greater than `_execution_index`, so `self._execution_index` - `self._max_execution_index` would result in a negative number.

Reply:
I think the name is supposed to be max_finished_execution_index. I will rename it